cloud/deployment/fromfile/fromfile.go

```diff
@@ -411,36 +625,36 @@ func deploymentExists(existingDeployments []astro.Deployment, deploymentNameToCr
 
 // deploymentFromName takes existingDeployments and deploymentName as its arguments.
 // It returns the existing deployment that matches deploymentName.
-func deploymentFromName(existingDeployments []astro.Deployment, deploymentName string) astro.Deployment {
+func deploymentFromName(existingDeployments []astroplatformcore.Deployment, deploymentName string) astroplatformcore.Deployment {
 	for i := range existingDeployments {
-		if existingDeployments[i].Label == deploymentName {
+		if existingDeployments[i].Name == deploymentName {
```

Comment:
i know this isn't part of this PR but 2 deployments with the exact same name can exist in the same workspace
Wouldn't this be a problem?

Reply:
Yeah this is old code but we should have any error to stop an update if their are two deployments with the same name